tests/intensive/sam_parity_tests.py

```
@@ -47,6 +54,20 @@ def _create_test_dataset(num_samples=5, seed=51):
     return dataset
 
 
+def _create_video_dataset(num_samples=2, seed=51):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_create_video_dataset`

(ANN202)